README.md

```diff
@@ -98,50 +98,58 @@ Polly provides a variety of resilience strategies. Alongside the comprehensive g
 
 Polly categorizes resilience strategies into two main groups:
 
-- **Reactive**: These strategies handle specific exceptions that are thrown, or results that are returned, by the callbacks executed through the strategy.
-- **Proactive**: Unlike reactive strategies, proactive strategies do not focus on handling errors by the callbacks might throw or return. They can make pro-active decisions to cancel or reject the execution of callbacks (e.g., using a rate limiter or a timeout resilience strategy).
+### Reactive
 
-| Strategy | Reactive | Premise | AKA | How does the strategy mitigate?|
-| ------------- | --- | ------------- |:-------------: |------------- |
-|**Retry** <br/>(strategy family)<br/><sub>([quick-start](#retry)&nbsp;;&nbsp;[deep](https://www.pollydocs.org/strategies/retry))</sub> |Yes|Many faults are transient and may self-correct after a short delay.| *Maybe it's just a blip* |  Allows configuring automatic retries. |
-|**Circuit-breaker**<br/>(strategy family)<br/><sub>([quick-start](#circuit-breaker)&nbsp;;&nbsp;[deep](https://www.pollydocs.org/strategies/circuit-breaker))</sub>|Yes|When a system is seriously struggling, failing fast is better than making users/callers wait.  <br/><br/>Protecting a faulting system from overload can help it recover. | *Stop doing it if it hurts* <br/><br/>*Give that system a break* | Breaks the circuit (blocks executions) for a period, when faults exceed some pre-configured threshold. |
-|**Timeout**<br/><sub>([quick-start](#timeout)&nbsp;;&nbsp;[deep](https://www.pollydocs.org/strategies/timeout))</sub>|No|Beyond a certain wait, a success result is unlikely.| *Don't wait forever*  |Guarantees the caller won't have to wait beyond the timeout. |
-|**Rate Limiter**<br/><sub>([quick-start](#rate-limiter)&nbsp;;&nbsp;[deep](https://www.pollydocs.org/strategies/rate-limiter))</sub>|No|Limiting the rate a system handles requests is another way to control load. <br/><br/> This can apply to the way your system accepts incoming calls, and/or to the way you call downstream services. | *Slow down a bit, will you?*  |Constrains executions to not exceed a certain rate. |
-|**Fallback**<br/><sub>([quick-start](#fallback)&nbsp;;&nbsp;[deep](https://www.pollydocs.org/strategies/fallback))</sub>|Yes|Things will still fail - plan what you will do when that happens.| *Degrade gracefully*  |Defines an alternative value to be returned (or action to be executed) on failure. |
-|**Hedging**<br/><sub>([quick-start](#hedging)&nbsp;;&nbsp;[deep](https://www.pollydocs.org/strategies/hedging))</sub>|Yes|Things can be slow sometimes, plan what you will do when that happens.| *Hedge your bets*  | Executes parallel actions when things are slow and waits for the fastest one.  |
+These strategies handle specific exceptions that are thrown, or results that are returned, by the callbacks executed through the strategy.
+
+| Strategy | Premise | AKA | Mitigation |
+| ------------- | ------------- |:-------------: |------------- |
```

Comment:
I think the third column looks a bit weird being centered when all the others are left-aligned.
```suggestion
| ------------- | ------------- | -------------- | ------------ |
```